src/lib.rs

```diff
@@ -9,11 +9,13 @@ use std::pin::Pin;
 use std::sync::Arc;
 
 use base64::prelude::{Engine, BASE64_URL_SAFE_NO_PAD};
-use hyper::client::connect::Connect;
-#[cfg(feature = "hyper-rustls")]
-use hyper::client::HttpConnector;
+use http_body_util::{BodyExt, Full};
+use hyper::body::{Bytes, Incoming};
 use hyper::header::{CONTENT_TYPE, LOCATION};
-use hyper::{Body, Method, Request, Response, StatusCode};
+use hyper::{Method, Request, Response, StatusCode};
+#[cfg(feature = "hyper-rustls")]
+use hyper_util::client::legacy::connect::HttpConnector;
+use hyper_util::client::legacy::{connect::Connect, Client as HyperClient};
```

Comment:
Tiny nit: let's keep these two on separate lines (one line per module like the rest of these).